opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,57 @@
+import os
+from google.cloud import monitoring
+from google.cloud.monitoring import Client
+from google.cloud.monitoring import Metric
+from google.cloud.monitoring import MetricDescriptor
+from google.cloud.monitoring import MetricKind, ValueType
+from google.cloud.monitoring import LabelValueType
+from google.cloud.monitoring import LabelDescriptor
+from google.cloud.monitoring import Resource
+from opencensus.stats import view
+from datetime import datetime
+from datetime import timedelta
+
+class StackDriverExporter(object):
+    def __init__(self, client=None, project_id=None, resource=None):
+        if client is None:
+            client = Client(project=project_id)
+
+        self.client = client
+        self.project_id = client.project
+        self.resource = client.resource('global', {})
+
+    def emit(self, views, datapoint=None):
+        name = 'projects/{}'.format(self.project_id)
```

Comment:
Move this line to `__init__` so you don't need to construct the name every time you call `emit`.